Scaffold docs for Intercom Web and Cloud Actions (#3511)
segmentio/segment-docs: src/connections/destinations/catalog/actions-intercom-cloud/index.md

- **Granular control over data sent.** You can customize the conditions under which the events are sent to Intercom.
- **Support for lead creation.** You can create contacts with a role of `lead`, associate them with a company, send events for them, and convert them to a `user`.

## Getting started

1. From the Segment web app, navigate to **Connections > Catalog**.
2. Search for **Intercom Cloud Mode (Actions)** in the Destinations Catalog, and select the destination.
3. Click **Configure Intercom Cloud Mode (Actions)**.
4. Select the source that will send data to Intercom Cloud Mode (Actions) and follow the steps to name your destination.
5. On the **Settings** tab, authenticate with Intercom using OAuth.
6. Follow the steps in the Destinations Actions documentation on [Customizing mappings](/docs/connections/destinations/actions/#customizing-mappings).
7. Enable the destination and configured mappings.

{% include components/actions-fields.html settings="true"%}

{% include components/actions-fields.html%}

> ## FAQ & Troubleshooting

Reviewer: ```suggestion
## FAQs 
```

### Why is a company I created missing from my Intercom dashboard?

> If a company is created without an attached user, the company will not appear on Intercom's dashboard. This is expected. Once a user is attached to the company, it should appear in the list of companies.

Reviewer: ```suggestion
If a company is created without an attached user, the company won't appear on Intercom's dashboard. This is expected. Once a user is attached to the company, the company appears in the list of companies.
```

### Why isn’t a user getting attached to a company?

> When using the Identify Company action, Segment will first create or update a company. In the same action, Segment will also attach the user in your group call to that company. If the user does not exist in Intercom when the action runs, we will create/update the company but not attach the user. Please ensure the user is created in Intercom first.

Reviewer: ```suggestion
When using the Identify Company action, Segment first creates or updates a company. In the same action, Segment also attaches the user in your group call to that company. If the user does not exist in Intercom when the action runs, Segment will create and update the company but not attach the user. Please ensure the user is created in Intercom first.
```
